frontends/concrete-python/examples/sha1/sha1.py

```diff
@@ -115,7 +115,7 @@ def add2(x, y):
 
         return ans
 
-    # ruff: noqa:N805
+    @staticmethod
     @fhe.function(
```

Comment:
(Not a blocker) @aPere3 can we make `fhe.function` enables the static method implicitly?